src/um/objidl.rs

```diff
@@ -1,30 +1,113 @@
-// Copyright © 2016-2017 winapi-rs developers
+// Copyright © 2015-2017 winapi-rs developers
```

Comment:
There's a conflict, and it's probably due to the removal of the copyright year lines.